ste/xfer-anyToRemote.go

```diff
@@ -81,11 +82,22 @@ func anyToRemote(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pacer, sen
 			return
 		}
 		if exists {
-			// logging as Warning so that it turns up even in compact logs, and because previously we use Error here
-			jptm.LogAtLevelForCurrentTransfer(pipeline.LogWarning, "File already exists, so will be skipped")
-			jptm.SetStatus(common.ETransferStatus.FileAlreadyExistsFailure()) // TODO: question: is it OK to always use FileAlreadyExists here, instead of BlobAlreadyExists, even when saving to blob storage?  I.e. do we really need a different error for blobs?
-			jptm.ReportTransferDone()
-			return
+			shouldOverwrite := false
+
+			// if necessary, prompt to confirm user's intent
+			if jptm.GetOverwriteOption() == common.EOverwriteOption.Prompt() {
+				// don't show the full URL, as it's too long and hard to read
+				parsed, _ := url.Parse(info.Destination)
+				shouldOverwrite = jptm.GetOverwritePrompter().shouldOverwrite(parsed.Path)
+			}
+
+			if !shouldOverwrite {
+				// logging as Warning so that it turns up even in compact logs, and because previously we use Error here
+				jptm.LogAtLevelForCurrentTransfer(pipeline.LogWarning, "File already exists, so will be skipped")
+				jptm.SetStatus(common.ETransferStatus.FileAlreadyExistsFailure())
```

Comment:
Alright here's the bit where I'm less familiar with-- Is this considered a failure or registered as a skip in azcopy's output? If it's considered a failure, I can see some confusion being generated.

Reply:
It's considered skipped, which I think John just changed to prevent such confusions.

Reply:
Ah. I feel like changing the name of things that would cause a skip but not a failure to have a suffix like Skipped instead of Failure (ex. `FileAlreadyExistsSkipped` instead of `FileAlreadyExistsFailure`) would be good for readability. Thoughts?

Reply:
Renamed accordingly.